estate/models/estate_property.py

```diff
@@ -0,0 +1,22 @@
+from odoo import fields, models
+
+
+class EstatePropertyModel(models.Model):
+    _name = "estate_property"
+    _description = "Defines the model of a real estate property"
+
+    name = fields.Char(required=True)
```

Comment:
Your fields looks good for now, just a small explanation on the labels of the fields:
Odoo tries to automatically compute them if there is none provided.
So `date_availability` will become `Date Availability` for example. If you want to provide one, you can do something like `date_availability = fields.Date(string="Date of availability")`